app/app.go

```diff
@@ -212,6 +217,7 @@ var (
 		claimtypes.ModuleName:           nil,
 		interchainquerytypes.ModuleName: nil,
 		icatypes.ModuleName:             nil,
+		auctionmoduletypes.ModuleName:   {authtypes.Minter, authtypes.Burner, authtypes.Staking},
```

Comment:
Why do we need mint, burn, staking perms for auction module account?

Reply:
We don't, good catch. This was an automatic update which the tool "ignite" made when I said I wanted the auction keeper to contain a bank keeper but it wasn't anything I was trying to add on purpose. We only need the ability to send tokens and receive tokens